### PR TITLE
Cover get_history_for_case endpoint

### PR DESCRIPTION
**Pull request by ervuks**
Cover get_history_for_case endpoint

### CODE REVIEW

File: lib/test_case_history.dart [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    this.createdOn,
+    this.id,
+    this.typeId,
+    this.userId,
+  });
+
+  factory _History.fromJson(Map<String, dynamic> json) {
+    DateTime? createdOn;
+
+    createdOn = DateTime.fromMillisecondsSinceEpoch(json['created_on']);
```
Comment (tshedor) [inline lib/test_case_history.dart:73]:
```suggestion
    var createdOn = DateTime.fromMillisecondsSinceEpoch(json['created_on']);
```

File: lib/test_case_history.dart [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    this.options,
+    this.typeId,
+  });
+
+  factory _Changes.fromJson(Map<String, dynamic> json) {
+    return _Changes(
+      field: json['field'],
+      label: json['label'],
+      oldText: json['old_text'],
+      oldValue: json['old_value'],
```
Comment (tshedor) [inline lib/test_case_history.dart:124]: Alpha

File: test/test_case_test.dart [hunk 3]
```diff
@@ -22,5 +23,11 @@ void main() {
       final result = await TestCase.getAll(1);
       expect(result.asJson, sampleForTestCases);
     });
+
+    test('#getHistory', () async {
```
Comment (tshedor) [inline test/test_case_test.dart:27]: This should be in its own test file (`test/test_case_history_test.dart`) and be annotated `.get` instead

**Review by tshedor — COMMENTED** [review]
Would you mind adding this to the README and example file as well please?

**Review by Victorrent — APPROVED** [review]

File: lib/test_case_history.dart [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+  _Changes({
+    this.field,
+    this.label,
+    this.oldText,
+    this.oldValue,
+    this.newText,
+    this.newValue,
+    this.options,
+    this.typeId,
```
Comment (Victorrent) [inline lib/test_case_history.dart:114]:
```suggestion
    this.field,
    this.label,
    this.newText,
    this.newValue,
    this.oldText,
    this.oldValue,
    this.options,
    this.typeId,
```

File: test/test_case_history_test.dart [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import 'package:test/test.dart';
+import 'package:test_rail_dart/test_case_history.dart';
+import '__helpers__.dart';
+import 'data/sample_test_case.dart';
+
+void main() {
+  group('TestCase', () {
```
Comment (Josiassejod1) [inline test/test_case_history_test.dart:7]:
```suggestion
  group('TestCaseHistory', () {
```

File: example/main.dart [hunk 6]
```diff
@@ -49,27 +49,5 @@ void main(List<String> args) async {
     statusId: 1,
   );
 
-  await newRun.update(
```
Comment (tshedor) [inline example/main.dart:52]: Should all of this really be deleted? I think we still want this example code.

**Review by tshedor — CHANGES_REQUESTED** [review]

**Review by tshedor — APPROVED** [review]

**Pull request merged**